sdk/textanalytics/Azure.AI.TextAnalytics/src/PiiTaskParameters.cs

```diff
@@ -15,7 +15,7 @@ public partial class PiiTaskParameters
         /// <summary>
         /// ModelVersion
         /// </summary>
-        public string ModelVersion { get; set; }
+        public string ModelVersion { get; set; } = "latest";
```

Comment:
Is the 'latest' default value necessary? If user use 'null' as the ModelVersion. does service side automatically use the default value (any value that service can use). If we set the default value in the SDK side, then we are using 'latest' keyword as the default value and user would always assume it is true. Even though that 'latest' is the correct to user. But we can remove this concern by having not set the default value in the SDK side, but pick whatever the default value from the service side.

Reply:
Also, StringIndexType is not explored publicly right now. Is that right? @maririos

Reply:
> Also, StringIndexType is not explored publicly right now. Is that right? @maririos

Correct. We want to have it private for no (internal for .NET)

Reply:
yeah so the model version needs to be set. If the model version property is not set it results in a 400. 

StringIndexType was not exposed intentionally. @maririos ^^^

Reply:
@mssfang this is a bug on the service for analyze endpoint, so in .NET we have it like this. Eventually it will be removed and respect whatever the user sends.
Are you not seeing this from Java?

Reply:
Java has the same problem. I didn't realize that I manually set 'latest' to all options's modelVersion.  Yeah. Agreed that  we should keep the defaultValue as 'latest' for now. So user have the better UX for this preview.